src/components/applicantMain/ApplicantMain.tsx

```diff
@@ -11,8 +11,13 @@ import ListItem from '@mui/material/ListItem'
 import ListItemButton from '@mui/material/ListItemButton'
 import ListItemIcon from '@mui/material/ListItemIcon'
 import ListItemText from '@mui/material/ListItemText'
-import InboxIcon from '@mui/icons-material/MoveToInbox'
 import MailIcon from '@mui/icons-material/Mail'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import EmojiPeopleIcon from '@mui/icons-material/EmojiPeople'
+import EmojiEventsIcon from '@mui/icons-material/EmojiEvents'
+import ExtensionIcon from '@mui/icons-material/Extension'
+import SupportAgentIcon from '@mui/icons-material/SupportAgent'
+import HomeRepairServiceIcon from '@mui/icons-material/HomeRepairService'
```

Comment:
lets change all imports in this file to style like this:

import { ExtensionIcon, EmojiEventsIcon, ALL_OTHER_ICONS } from '@mui/icons-material'

Reply:
throws an error

export 'AccountCircleIcon' (imported as 'AccountCircleIcon') was not found in '@mui/icons-material'